test/verify/check-storage-hidden

```diff
@@ -123,6 +126,31 @@ class TestStorageHidden(StorageCase):
         b.wait_visible("#storage")
         self.assertEqual(m.execute(f"grep {mount_point_2} /etc/fstab || true"), "")
 
+    @unittest.skipUnless("ubuntu" in testvm.DEFAULT_IMAGE, "Only test snaps on Ubuntu")
+    def testHiddenSnap(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        # We expect there to be at least one loop back device mounted
+        # somewhere below /snap.
+        snap_loops = []
+        devices = json.loads(m.execute("lsblk --json --list --output NAME,TYPE,MOUNTPOINT"))
+        for d in devices["blockdevices"]:
+            if d["type"] == "loop" and d["mountpoint"].startswith("/snap/"):
+                snap_loops.append(d["name"])
```

Comment:
There should be either an `else:` for the `for` loop or an assertion that `snap_loops` is not empty.

Reply:
Right, good point!

Reply:
Done.